pages/[id]/bridge.tsx

```diff
@@ -101,20 +132,66 @@ const Bridge = () => {
   };
 
   const handleBridge = () => {
-    return bridgeTarget === BridgeTarget.L2 ? bridgeToL2!() : bridgeToL1!();
+    if (bridgeTarget === BridgeTarget.L2) {
+      if (!bridgeToL2) throw new Error('bridgeToL2 is not defined');
+      bridgeToL2();
+    } else {
+      if (!bridgeToL1) throw new Error('bridgeToL1 is not defined');
+      bridgeToL1();
+    }
   };
 
-  const source =
-    bridgeTarget === BridgeTarget.L2
-      ? { ...l1Config, ...l1, fee: fees?.l1 || BigInt(0) }
-      : { ...l2Config, ...l2, fee: fees?.l2 || BigInt(0) };
-  const target =
+  const formatError = (e: Error | null): ErrorReturnType => {
```

Comment:
Nit: I think this function could be abstracted out to a utils file.

Reply:
Generally agree, but currently this function uses state from the component (aka `source.token?.symbol`, `chain?.nativeCurrency.symbol`, `bridgeToL1Error`, etc. For now i'll leave as is.